keyboards/keychron/q9_plus/readme.md

```diff
@@ -0,0 +1,21 @@
+# Keychron Q9 Plus
+
+![Keychron Q9 Plus](https://i.imgur.com/Yp9K7Gs.jpg)
+
+A customizable 40% keyboard.
+
+* Keyboard Maintainer: [Keychron](https://github.com/keychron)
+* Hardware Supported: Keychron Q9 Plus
+* Hardware Availability:[Keychron Q9 QMK Custom Mechanical Keyboard](https://www.keychron.com/products/keychron-q9-qmk-custom-mechanical-keyboard)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make keychron/q9_plus/ansi_encoder:default
+
+Flashing example for this keyboard ([after setting up the bootloadHID flashing environment](https://docs.qmk.fm/#/flashing_bootloadhid))
```

Comment:
Uhhh, I don't think this is correct.  Bootloader is configured for stm32-dfu, is it not?

Reply:
Yes, it is. the no need information has been deleted now, thanks!